### ISSUE TITLE
Remote support add-on does not appear in the tools menu

### ISSUE THREAD

**Issue by jcoffin1** [nvaccess/nvda#8084]
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. When remote support add-on is enabled, the tools menu in the NVDA menu does not display the remote support option in tools. 

1. if you have the remote add-on installed, and enabled go to NVDA menu, arrow to tools, open tools menu, and remote support should be an option, but it's not there.

**Comment by josephsl** [comment]
Hi, are you perhaps running the latest next branch snapshot? If so, I raised an issue on Remote add-on regarding this matter. The ultimate culprit is a missing attribute for wx.Menu in wxPython 4. CC @TSpivey, @CToth.

**Comment by jcoffin1** [comment]
Yes I am. Using the latest next snapshot.
On March 14, 2018 10:50:42 AM Joseph Lee <notifications@github.com> wrote:>

**Comment by leonardder** [comment]
Strictly spoken, this is not a bug in NVDA itself, it should be solved in NVDA Remote. Therefore I'm temped to close this in favour of nvdaremote/nvdaremote#170 and nvdaremote/nvdaremote#171.

**Comment by DrSooom** [comment]
Isn't this fixed in NVDA 2018.4.1 and NVDA Remote Access 2.2?

**Comment by ctoth** [comment]
Yes, this should be closed.

**Comment by Adriani90** [comment]
Closing as works for me.
